src/MediaPlayer/MPNowPlayingInfoCenter.cs

```diff
@@ -55,6 +65,27 @@ internal NSDictionary ToDictionary ()
 				dict.Add (MPNowPlayingInfoCenter.PropertyChapterNumber, new NSNumber (ChapterNumber.Value));
 			if (ChapterCount.HasValue)
 				dict.Add (MPNowPlayingInfoCenter.PropertyChapterCount, new NSNumber (ChapterCount.Value));
+
+			// iOS 10
+			if (DefaultPlaybackRate.HasValue)
+				dict.Add (MPNowPlayingInfoCenter.PropertyDefaultPlaybackRate, new NSNumber (DefaultPlaybackRate.Value));
```

Comment:
did you run your test on iOS 9.3 ? 

e.g. if `MPNowPlayingInfoCenter.PropertyDefaultPlaybackRate` is null on iOS 9.3 then you likely get an `ArgumentNullException` here.

The fact that the field are not (yet) decorated makes it even more likely to occur. In any case some protection can be used, e.g.

```
if (DefaultPlaybackRate.HasValue) {
    var key = MPNowPlayingInfoCenter.PropertyDefaultPlaybackRate;
    if (key != null)
       dict.Add (key, ...);
}```